package.json

```diff
@@ -1,6 +1,7 @@
 {
     "private": true,
     "name": "public-website",
+    "homepage": "http://csclub.uwaterloo.ca/~sesoc/",
```

Comment:
Would there a difference if we update this to `https`?

Reply:
I'll take a look